docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +340,71 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Proxy Jump in Trusted Clusters?">
+  
+Normally, you connect to leaf cluster SSH nodes through the root cluster proxy,
+which is provided in the `ProxyCommand` line of the SSH config generated above.
+
+In scenarios where leaf proxies are reachable by SSH clients, you might prefer
+to connect directly through the leaf proxies to avoid potential latency issues
+of the connection being routed through the root cluster.
+
+To support this scenario, the `tsh proxy ssh` command provides the `-J` flag.
+This flag points at a cluster's web proxy address and indicates that the
+OpenSSH client will be connecting through that proxy instead of the proxy
+you're currently logged into:
+
+```txt
+Host *.{{ .NodeName }}.{{ .ClusterName }}
+   Port 3022
+   ProxyCommand tsh proxy ssh -J proxy.leaf1.example.com:443 %r@%h:%p
+```
+
+The jump proxy address and the node name to connect to can be dynamically
+determined from the SSH config's full hostname spec (`%h:%p`) using `{{proxy}}`
+template variable:
+
+```txt
+Host *.example.com
+   Port 3022
+   ProxyCommand tsh proxy ssh -J {{proxy}} %r@%h:%p
+```
+
+The rules of how to parse the proxy address and node name from the full host
+spec can be defined in the `tsh` configuration file (`~/.tsh/config/config.yaml`
+or a global `/etc/tsh.yaml`):
+
+```yaml
+proxy_templates:
+- template: '^(\w+)\.(leaf1\.example\.com):(.*)$'
+  proxy: "$2:443"
+- template: '^(\w+)\.(leaf2\.example\.com):(.*)$'
+  proxy: "$2:3080"
+  host: "$1"
+```
+
+| Field      | Description |
+| ---------- | ----------- |
+| `template` | (Required) Regular expression full SSH config host spec is matched against. |
+| `proxy`    | (Required) Web proxy address to use. Can reference regex groups from `template`. |
+| `host`     | (Optional) Node name to use. Can reference regex groups from `template`. Defaults to full host spec. |
+
+Given the configuration above, the following command will connect to the node
```

Comment:
I think we should use "host" instead of "node" when talking about servers that are running `openssh` but not Teleport. This way, there's no risk of confusion with Teleport Nodes.

Reply:
Actually in this example these are only Teleport nodes. I am adding an example of connecting to an openSSH host though.